multi_block_core.dic

```diff
@@ -124,19 +144,20 @@ save_DIFFRN_RADIATION
 ;
     _name.category_id             DIFFRACTION
     _name.object_id               DIFFRN_RADIATION
-    _category_key.name            '_diffrn_radiation.diffrn_id'
+    _category_key.name            '_diffrn_radiation.id'
 
 save_
 
 save_diffrn_radiation.diffrn_id
```

Comment:
Maybe this item should be removed altogether instead of deprecating it?

Arguments in favour of removal:
1. This item was only recently added to this dictionary.
2. There are already data items with the same name in the mmCIF and the imgCIF dictionaries so deprecating them here might lead to some confustion.
3. The item is no longer referenced as a category key so the intended use case in general is a bit unclear.

Reply:
I've removed the definition as suggested.